test/sql/test_materialized_view/T/test_mv_swap

```diff
@@ -1,5 +1,7 @@
 -- name: test_mv_swap
 
+admin set frontend config('enable_mv_automatic_active_check'='false');
```

Comment:
should this case be marked as @sequential ? so the change of frontend configuration won't affect other cases that may be running in the same time?